pyvrp/cpp/Individual.cpp

```diff
@@ -186,6 +179,6 @@ std::ostream &operator<<(std::ostream &out, Individual const &indiv)
         out << '\n';
     }
 
-    out << "Cost: " << indiv.cost() << '\n';
+    out << "Distance: " << indiv.distance << '\n';
```

Comment:
@leonlan do you know if this is allowed? (I'm not sure if VRPLIB requires a particular word / value here, or if we can do whatever we want)

Reply:
I don't mean the package, I mean the format specification, if any exists

Reply:
Ah OK, then we can do whatever. Thanks!